content/docs/command-reference/fetch.md

```diff
@@ -17,22 +17,12 @@ positional arguments:
 
 ## Description
 
-Downloads DVC-tracked files from remote storage into the cache of the project
-(without placing them in the <abbr>workspace</abbr>, like `dvc pull` would).
-This makes them available for linking (or copying) into the workspace (refer to
-[`dvc config cache.type`](/doc/command-reference/config#cache)).
+Downloads tracked data from remote storage into the <abbr>cache</abbr> (without
```

Comment:
it's better to be neutral (data almost equals to datasets) - files and directories is better

Reply:
Changed 👍